x/emissions/keeper/msgserver/msg_server_losses.go

```diff
@@ -31,12 +31,9 @@ func (ms msgServer) InsertBulkReputerPayload(
 	}
 
 	// Check if the topic exists
-	topicExists, err := ms.k.TopicExists(ctx, msg.TopicId)
+	topic, err := ms.k.GetTopic(ctx, msg.TopicId)
 	if err != nil {
-		return nil, err
-	}
-	if !topicExists {
-		return nil, types.ErrTopicDoesNotExist
+		return nil, sdkerrors.ErrNotFound
```

Comment:
this isn't the only error in the world... say if we change GetTopic later...

why remove the `if !topicExists` conditional?

Reply:
yeah I'm basically saving one query since we will need the `topic.AlphaRegret` later in the function.